src/liberror/specification

```diff
@@ -660,3 +660,58 @@ description:python warning
 severity:warning
 ingroup:plugin
 module:python
+
+number:113
+description:couldnt open file.
+severity:error
+ingroup:plugin
+module:lineendings
+
+number:114
+description:invalid line ending.
+severity:error
+ingroup:plugin
+module:lineendings
+
+number:115
+description:inconsistent line endings
+severity:error
+ingroup:plugin
+module:lineendings 
+
+number:116
+description: couldn't open file.
+severity:error
+ingroup:plugin
+module:csvstorage
+
+number:117
+description:invalid number of columns
+severity:error
+ingroup:plugin
+module:csvstorage
+
+number:118
+description:see Reason.
+severity:warning
+ingroup:plugin
+module:csvstorage
+
+number:119
+description:out of memory.
```

Comment:
I am not against an error for out of memory. But the error should be not specific to this plugin and should be used consistently (for every allocation) in the plugins which are out-of-memory safe.